lib/matplotlib/axes/_axes.py

```diff
@@ -3050,11 +3050,43 @@ def boxplot(self, x, notch=False, sym=None, vert=True, whis=1.5,
 
         .. plot:: mpl_examples/statistics/boxplot_demo.py
         """
+        # If defined in matplotlibrc, apply the value from rc file
+        # Overridden if argument is passed
+        if rcParams['boxplot.whiskers'] != 1.5:
+            whis = rcParams['boxplot.whiskers']
```

Comment:
If I understand this correctly, then a bad situation can arise. If someone has an rc file with a different value for `boxplot.whiskers`, but explicitly passes in a value for `whis`, then the rcfile's value takes precedence. That would be incorrect. Why not set the default value of `whis` to `None` and test for that?

I have similar concerns for `bootstrap` because it completely ignores the possibility that the user explicitly sets some value for it.